qucs/schematic.cpp

```diff
@@ -15,626 +15,641 @@
  *                                                                         *
  ***************************************************************************/
 
-#include <stdlib.h>
 #include <limits.h>
+#include <stdlib.h>
 
-#include <QFileInfo>
-#include <QPrinter>
-#include <QPaintDevice>
+#include <QAction>
+#include <QApplication>
+#include <QClipboard>
+#include <QDebug>
 #include <QDir>
-#include <QTextStream>
-#include <QDragLeaveEvent>
-#include <qt3_compat/qt_compat.h>
-#include <QPixmap>
 #include <QDragEnterEvent>
+#include <QDragLeaveEvent>
 #include <QDragMoveEvent>
 #include <QDropEvent>
-#include <QMouseEvent>
 #include <QEvent>
-#include <QWheelEvent>
-#include <QPainter>
-#include <QAction>
+#include <QFileInfo>
 #include <QLineEdit>
-#include <QUrl>
 #include <QListWidget>
-#include <QDebug>
-#include <QApplication>
-#include <QClipboard>
+#include <QMouseEvent>
+#include <QPaintDevice>
+#include <QPainter>
+#include <QPixmap>
+#include <QPrinter>
+#include <QTextStream>
+#include <QUrl>
+#include <QWheelEvent>
+#include <qt3_compat/qt_compat.h>
 
-#include "qucs.h"
-#include "schematic.h"
+#include "components/vafile.h"
+#include "components/verilogfile.h"
+#include "components/vhdlfile.h"
+#include "diagrams/diagrams.h"
 #include "main.h"
+#include "mouseactions.h"
 #include "node.h"
+#include "paintings/paintings.h"
+#include "qucs.h"
+#include "schematic.h"
 #include "textdoc.h"
 #include "viewpainter.h"
-#include "mouseactions.h"
-#include "diagrams/diagrams.h"
-#include "paintings/paintings.h"
-#include "components/vhdlfile.h"
-#include "components/verilogfile.h"
-#include "components/vafile.h"
 
 #include "misc.h"
 
 // just dummies for empty lists
-Q3PtrList<Wire>      SymbolWires;
-Q3PtrList<Node>      SymbolNodes;
-Q3PtrList<Diagram>   SymbolDiags;
+Q3PtrList<Wire> SymbolWires;
+Q3PtrList<Node> SymbolNodes;
+Q3PtrList<Diagram> SymbolDiags;
 Q3PtrList<Component> SymbolComps;
 
-
-Schematic::Schematic(QucsApp *App_, const QString& Name_)
+Schematic::Schematic(QucsApp *App_, const QString &Name_)
     : QucsDoc(App_, Name_)
 {
-  symbolMode = false;
```

Comment:
Indentation should be preserved here. Otherwise git shows 100% code replacement, which make hard to track the changes.